Issue/359/dsps v3.0.0

torluca opened the pull request.

eacharles commented:
I pulled in the changes from main, so that there aren't conflicts anymore.

But we probably want to figure out something else to do about the files in 
src/rail/examples_data/creation_data/data/dsps_default_data  it looks like those are 59 MB, which seems really big for a code repo.

We have some examples use setting up examples to curl data from NERSC, we could do that here.

torluca commented:
Hi @eacharles , I followed your suggestions and implemented the following changes:
- pull latest changes from main
- curl ssp templates from public lsst directory and removed the templates in the examples_data folder
- fix the issues with the unit tests
- removed the dsps_reqs.txt file, since now dsps is pip installable and does not require diffstar and diffmah to be git cloned
If you think that everything is alright, then you can approve the review :)
Grazie mille!

eacharles reviewed (approved):
Thanks Luca.
